pypika/tests/test_inserts.py

```diff
@@ -88,6 +88,35 @@ def test_insert_null(self):
         self.assertEqual('INSERT INTO "abc" VALUES (NULL)', str(query))
 
 
+class InsertIntoReturningTests(unittest.TestCase):
```

Comment:
These test cases look pretty thorough and I'm not sure what's missed but coveralls is reporting 0.5% drop in test case coverage. Can you please check and make sure all of your conditions are covered by test cases?